superset-frontend/src/chart/Chart.jsx

```diff
@@ -201,7 +201,10 @@ class Chart extends React.PureComponent {
         showMessage={false}
       >
         <Styles className="chart-container">
-          <div className={`slice_container ${isFaded ? ' faded' : ''}`}>
+          <div
+            className={`slice_container ${isFaded ? ' faded' : ''}`}
```

Comment:
this is old code, but maybe we can refactor this with `classnames` (cx)

Reply:
There are probably a million of these in the codebase... wonder if it's worth tasking someone to go on a seek and destroy mission throughout the codebase, to better establish the pattern.